api/v1alpha1/envoypatchpolicy_types.go

```diff
@@ -59,7 +59,7 @@ type EnvoyPatchPolicySpec struct {
 	// the priority i.e. int32.min has the highest priority and
 	// int32.max has the lowest priority.
 	// Defaults to 0.
-	Priority int32 `json:"priority"`
+	Priority int32 `json:"priority,omitempty"`
```

Comment:
this's OK to me, but I recall the api guide said should be a pointer?

Reply:
its still required internally, `omitempty` allows the user to skip setting it when creating the resource and use the default value / `0` instead